Package.swift

```diff
@@ -48,7 +48,7 @@ let packageDependencies: [Package.Dependency] = [
   ),
   .package(
     url: "https://github.com/apple/swift-protobuf.git",
-    from: "1.19.0"
+    from: "1.20.0"
```

Comment:
Can we use 1.20.1? (1.20.0 has warnings and there's no harm in setting to the newest)